packages/beeq/src/components/input/scss/bq-input.scss

```diff
@@ -5,20 +5,20 @@
 @import './bq-input.variables';
 
 :host {
-  @apply block w-full;
+  @apply block is-full;
 }
 
 /* -------------------------------------------------------------------------- */
 /*                            Label and Helper text                           */
 /* -------------------------------------------------------------------------- */
 
 .bq-input--label {
-  @apply mb-[--bq-input--label-margin-bottom] flex flex-grow items-center gap-[--var(--bq-input--gap-label)];
```

Comment:
I have removed this rule `gap-[--var(--bq-input--gap-label)];` because it is unnecessary in our context.